client/blocks/term-tree-selector/terms.jsx

```diff
@@ -3,10 +3,8 @@
  *
  * @format
```

Comment:
Want to pull `@format` out of external deps?

I've looked over this file and smoke tested via the post editor. Looks good 👍 

(a review in parts)